packages/ember-htmlbars/lib/main.js

```diff
@@ -79,7 +67,6 @@ registerHelper('textarea', textareaHelper);
 registerHelper('collection', collectionHelper);
 registerHelper('each', eachHelper);
 registerHelper('unbound', unboundHelper);
-registerHelper('concat', concat);
```

Comment:
Why remove `concat` helper?  Because its original purpose was replaced with the hook?